docs/framework/migration-guide/net-framework-4-migration-issues.md

```diff
@@ -296,7 +296,7 @@ Namespaces: <xref:System.Xml.Linq>; <xref:System.Xml.Schema>, <xref:System.Xml.X
 | **Namespace attribute** | To prevent data corruption, an <xref:System.Xml.XPath.XPathNavigator> object now returns the local name of the `x:xmlns` attribute correctly. |
 | **Namespace declarations** | An <xref:System.Xml.XmlReader> object on a sub-tree no longer creates duplicate namespace declarations within one XML element. |
 | **Schema validation** | To prevent erroneous schema validation, the <xref:System.Xml.Schema.XmlSchemaSet> class allows for XSD schemas to be compiled correctly and consistently. These schemas can include other schemas; for example, `A.xsd` can include `B.xsd`, which can include `C.xsd`. Compiling any one of these causes this graph of dependencies to be traversed. |
-| **Script functions** | The [function-available function](https://msdn.microsoft.com/library/ms256124(v=vs.100).aspx) no longer incorrectly returns `false` when the function is actually available. |
+| **Script functions** | The [function-available function](https://msdn.microsoft.com/library/ms256124(v=vs.110).aspx) no longer incorrectly returns `false` when the function is actually available. |
```

Comment:
this is odd. it seems this content should be redirecting to docs but it's not. can you please open an issue for me to investigate this?